library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -47,36 +51,13 @@ class ChuckerInterceptor @JvmOverloads constructor(
     @Suppress("LongMethod", "ComplexMethod")
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
-        val requestBody = request.body()
-
-        val transaction = HttpTransaction()
-        transaction.requestDate = System.currentTimeMillis()
-        transaction.method = request.method()
-        transaction.populateUrl(request.url().toString())
-        transaction.setRequestHeaders(request.headers())
-        transaction.requestContentType = requestBody?.contentType()?.toString()
-        transaction.requestContentLength = requestBody?.contentLength() ?: 0L
 
-        val encodingIsSupported = io.bodyHasSupportedEncoding(request.headers().get("Content-Encoding"))
-        transaction.isRequestBodyPlainText = encodingIsSupported
-
-        if (requestBody != null && encodingIsSupported) {
-            val source = io.getNativeSource(Buffer(), io.bodyIsGzipped(request.headers().get("Content-Encoding")))
-            val buffer = source.buffer()
-            requestBody.writeTo(buffer)
-            var charset: Charset = UTF8
-            val contentType = requestBody.contentType()
-            if (contentType != null) {
-                charset = contentType.charset(UTF8) ?: UTF8
-            }
-            if (io.isPlaintext(buffer)) {
-                val content = io.readFromBuffer(buffer, charset, maxContentLength)
-                transaction.requestBody = content
-            } else {
-                transaction.isResponseBodyPlainText = false
-            }
-        }
+        return if (httpFeature.enabled) captureTransaction(request, chain)
```

Comment:
What do you think about keeping the same style for `if else` everywhere? 
For example, in the same file there are lines the conditions have braces for `if` and `else` cases (like on line 127).
I believe that it could improve readability a little bit.

Reply:
:+1:

Reply:
We can enable the `MandatoryBracesIfStatements` detekt rule if you think it's the case